### DISCUSSION TITLE
Taskbar items ordering individually

### DISCUSSION THREAD

**Discussion by robozdog** [valinet/ExplorerPatcher#2954] [Ideas]
When same instance apps are open in taskbar, they can only be ordered in group. It would be nice to order them individually. 
Using Never Combine option so each instance is own tab.

**Comment by pyrates999** [comment]
Install 7+ Taskbar Tweaker.  It has that feature.

**Comment by pyrates999** [comment]
You need to set the taskbar to windows 10.  Then you can use 7+ taskbar tweaker.
